examples/ex1.cpp

```diff
@@ -198,8 +198,15 @@ int main(int argc, char *argv[])
    }
```

Comment:
Add an example run for non-tensor CEED under `Device sample runs` in the documentation for this example, maybe
```
ex1 -m ../data/beam-tet.mesh -pa -d cuda
```